bootstrap/internal/cloudinit/cloudinit.go

```diff
@@ -121,3 +124,37 @@ func generate(kind string, tpl string, data interface{}) ([]byte, error) {
 
 	return out.Bytes(), nil
 }
+
+func cleanupAdditionalCloudInit(cloudInitData string) (string, error) {
+	m := make(map[string]interface{})
+
+	err := yaml.Unmarshal([]byte(cloudInitData), m)
+	if err != nil {
+		return "", err
```

Comment:
is it possible to add some context to errors here with `fmt.Errorf`, I expect at some point someone will pass an invalid yaml, and it's better to let users know more about what happened.

Reply:
done!